interfaces/dotnet/Cantera/src/SpeciesCollection.cs

```diff
@@ -80,23 +80,23 @@ internal unsafe SpeciesCollection(ThermoPhaseHandle handle)
     {
         _handle = handle;
 
-        var count = (int) InteropUtil.CheckReturn(LibCantera.thermo_nSpecies(handle));
+        var count = (int) InteropUtil.CheckReturn(LibCantera.thermo3_nSpecies(handle));
```

Comment:
Since you have standardized on `int` in CLib, you can remove the cast.

```suggestion
        var count = InteropUtil.CheckReturn(LibCantera.thermo3_nSpecies(handle));
```